internal/local/cmd.go

```diff
@@ -184,8 +184,23 @@ func New(provider k8s.Provider, portHTTP int, opts ...Option) (*Command, error)
 				cmd = exec.Command("open", url)
 			case "windows":
 				cmd = exec.Command("cmd", "/c", "start", url)
+			case "linux":
```

Comment:
While this looks pretty reasonable, I don't know all of the cross-platform methods (that we care about) to implement this, so perhaps its worth delegating this to a library like https://github.com/cli/browser

Reply:
I migrated to using that repo, first I'd seen it.  Thanks for the referral.